packages/gasket-plugin-swagger/lib/index.js

```diff
@@ -108,6 +108,30 @@ module.exports = {
         app.use(apiDocsRoute, swaggerUi.serve, swaggerUi.setup(swaggerSpec, ui));
       }
     },
+    /**
+     * Serve the Swagger Docs UI.
+     *
+     * @param {object} gasket - Gasket API
+     * @param {object} app - Fastify app instance
+     * @async
+     */
+    fastify: {
+      timing: {
+        before: ['@gasket/plugin-nextjs']
+      },
+      handler: async function fastify(gasket, app) {
+        const { swagger, root } = gasket.config;
+        const { ui = {}, apiDocsRoute, definitionFile } = swagger;
+
+        const swaggerSpec = await loadSwaggerSpec(root, definitionFile, gasket.logger);
+
+        app.register(require('@fastify/swagger'), {
```

Comment:
👍 Can you also delay the `require('swagger-ui-express')` to within the `express` lifecycle handler? It doesn't need to be loaded if it's not used.